### PR TITLE
Receipt Moblify

### LINKED ISSUE #14: buildout > impact receipt

### PR DESCRIPTION
**Pull request by nellbhealy**
Make receipt section mobile-friendly

### CODE REVIEW

File: src/components/receipt-view/data-section/DataBreakdownAggregate.js [hunk 1]
```diff
@@ -51,12 +51,17 @@ const DataBreakdownAggregate = ({
         windowDimensions={windowDimensions}
       />
       <div className={styles.impactText}>
-        <Typography variant="subtitle1">{text[type].title}</Typography>
-        <Typography variant="body2" className={styles.body}>
-          {width && (width - min) / width >= minRatio.body
-            ? text[type].body
-            : null}
-        </Typography>
```
Comment (willyd332) [inline src/components/receipt-view/data-section/DataBreakdownAggregate.js:59]: Is there a way to do this with just CSS? 
With an @media on a new classname?

But also, this works and we're on the last week, I don't think it's worth changing now so we can start QA on the receipt.

Comment (nellbhealy) [reply src/components/receipt-view/data-section/DataBreakdownAggregate.js:59]: Agreed. I definitely think there are more elegant ways to do this, but given that we need it to be done I think we should leave it as it.

File: src/components/receipt-view/data-section/DataSection.js [hunk 2]
```diff
@@ -61,12 +61,12 @@ const DataSection = ({ cartState }) => {
...
       container
       alignItems="center"
-      sm={ratio >= minRatio.body ? 12 : 10}
+      xs={ratio >= minRatio.body ? 12 : 10}
       spacing={1}
       className={styles.dataSectionBox}>
       <Grid
         item
-        sm={(width - min) / width >= minRatio.body ? 12 : 10}
+        xs={(width - min) / width >= minRatio.body ? 12 : 10}
```
Comment (willyd332) [inline src/components/receipt-view/data-section/DataSection.js:69]: Wait, I don't understand what this is doing. Will you explain it to me?
Couldn't the same thing be achieved by doing: 
xs={10}
sm={12}

**Review by willyd332 — APPROVED** [review]

File: src/components/receipt-view/data-section/DataSection.js (hunk 2 above)
Comment (nellbhealy) [reply src/components/receipt-view/data-section/DataSection.js:69]: Yeah it totally could. I think when I did this I didn't understand what the `xs` vs `sm` was really doing. That's a quick enough change, I'll go ahead and do that now before we merge it

Comment (nellbhealy) [reply src/components/receipt-view/data-section/DataSection.js:69]: So this is actually more precise than using the breakpoints, I think we should keep it for now? I spent a bit of time on it and couldn't get it to work as well as it does now. Might not be worth fixing for Version 1

Comment (willyd332) [reply src/components/receipt-view/data-section/DataSection.js:69]: Sounds good

**Pull request merged**